### ISSUE TITLE
Using outputs

### ISSUE THREAD

**Issue by karstenmueller** [pendrica/chef-provisioning-azurerm#24]
How could I use “outputs”? Would be handy to show an external IP for example. Amending the ARM template does not do it:

```json
    "outputs": {
        "externalIP" : {
            "type" : "string",
            "value": "[concat('external IP',reference(resourceId('Microsoft.Network/publicIPAddresses', 'lbex-ip-ext')))]"
        }
    },
```

**Comment by stuartpreston** [comment]
Currently the outputs are not retrieved from the deployment but we should show them somewhere.  Where would you like the output to be shown? In the chef client stdout?  I will take this as a suggestion and look to add it when I can!

**Comment by karstenmueller** [comment]
Yes, chef-client stdout will be best. Looking forward to see this enhancement progress:)

**Comment by stuartpreston** [comment]
I did some investigation on this tonight and it looks like the outputs are not returned when the deployment is completed in the Azure SDK for Ruby.  They seem to be part of a DeploymentPropertiesExtended object http://www.rubydoc.info/gems/azure_mgmt_resources/0.5.0/Azure/ARM/Resources/Models/DeploymentPropertiesExtended 

I will keep investigating.

**Comment by FedorRomanov** [comment]
Hello guys,
any update from MDST on how to get DeploymentPropertiesExtended object?
